batch/data-plane/src/test/java/com/microsoft/azure/batch/TaskTests.java

```diff
@@ -375,10 +412,13 @@ public void testGetTaskCounts() throws Exception {
             behaviors.add(option);
             batchClient.taskOperations().createTasks(jobId, tasksToAdd, behaviors);
 
-            TimeUnit.SECONDS.sleep(30);
+            //The Waiting period is only needed in record mode.
+            if(isRecordMode()) {
+                TimeUnit.SECONDS.sleep(30);
```

Comment:
is possible use threadSleepInRecordMode?

Reply:
Yes, 
Updated it.